docs/HowTo/Configure/Block-Proposal-Permissioning.md

```diff
@@ -0,0 +1,256 @@
+---
+description: Block proposal permissioning
+---
+
+# Block proposal permissioning
+
+!!! important
+
+    Only private networks using the [QBFT consensus protocol] support block proposal permissioning.
+
+    Block proposal permissioning is an early access feature, and functionality and options may be updated between releases.
+
+You can configure [block proposal permissioning](../../Concepts/PKI.md#block-proposal-permissioning)
+to ensure authorized validator nodes can propose new blocks in the network.
```

Comment:
Might be redundant, thinking we should add only this this. i.e. "to ensure only authorized validator nodes can propose new blocks in the network."